.storybook/webpack.config.js

```diff
@@ -14,12 +15,24 @@ module.exports = ({ config, mode }) => {
     ],
   });
 
+  config.module.rules.push({
```

Comment:
We can push this rule in the first `push` call (`push(1, 2)`), it'll be easier to understand the configuration if it's at the same location.